.github/workflows/matrix-bot.yml

```diff
@@ -3,9 +3,12 @@ on:
   pull_request_target:
     types: [opened, ready_for_review, closed]
 
+
 jobs:
   new-pr:
-    if: github.event.action == 'opened' && github.repository == 'ariel-os/ariel-os'
+    if: |
+      github.event.action == 'opened' && github.repository == 'ariel-os/ariel-os' &&
+      !contains(github.even.pull_request.labels.*.name, 'renovate')
```

Comment:
```suggestion
      !contains(github.event.pull_request.labels.*.name, 'renovate')
```

Reply:
Fixed in [550ac2d](https://github.com/ariel-os/ariel-os/pull/1249/commits/550ac2d7051db7dc7c50614cb1dba6715517c07f)